semantic-conventions/syntax.md

```diff
@@ -174,7 +174,6 @@ An attribute is defined by:
 - `sampling_relevant`, optional EXPERIMENTAL boolean,
```

Comment:
is it still considered experimental?

Reply:
I don't know, but they can stay experimental as far as I am concerned - all semantic conventions are experimental.